tasks/users.yaml

```diff
@@ -1,10 +1,12 @@
 ---
 - name: Create users and database grants
+  when: mariadb.galera.cluster == False or mariadb.galera.initializer == True
   mysql_user:
     name: "{{ username }}"
     password: "{{ password }}"
     host: "{{ host }}"
     priv: "{{ priv }}"
+    login_unix_socket: "{{ mariadb.socket if mariadb.socket else null }}"
```

Comment:
I wanted to use None, as it is standard in Python, but Ansible does not accept None. So that was the only one I knew of. I didn't know of none, thanks for pointing that out.